src/main/scala/org/apache/mesos/chronos/scheduler/jobs/Jobs.scala

```diff
@@ -110,7 +113,8 @@ case class ScheduleBasedJob(
                              @JsonProperty override val shell: Boolean = true,
                              @JsonProperty override val arguments: Seq[String] = List(),
                              @JsonProperty override val softError: Boolean = false,
-                             override val dataProcessingJobType: Boolean = false)
```

Comment:
not your code, but could we add @JsonProperty to dataProcessingJobType while you are at it. Not sure why this was left off.